### PR TITLE
HTS-955: Use build.sbt format

### PR DESCRIPTION
**Pull request by sureshhmrc**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+  ws,
+  "org.typelevel" %% "cats" % "0.9.0",
+  "uk.gov.hmrc" %% "bootstrap-play-25" % "1.4.0",
+  "com.github.kxbmap" %% "configs" % "0.4.4"
+)
+
+def testDependencies(scope: String = "test,it") = Seq(
+  "uk.gov.hmrc" %% "hmrctest" % "3.0.0" % scope,
+  "org.scalatest" %% "scalatest" % "3.0.4" % scope,
+  "org.pegdown" % "pegdown" % "1.6.0" % scope,
```
Comment (AndyHWChung) [inline build.sbt]: do we need pegdown?

Comment (sureshhmrc) [reply build.sbt]: ah, missed it again, let me fix it

Comment (sureshhmrc) [reply build.sbt]: fixed !

**Comment by AndyHWChung** [comment]
LGTM

**Pull request merged**